src/components/Footer/Footer.js

```diff
@@ -9,88 +10,71 @@ import logo from '../../images/white_logo.png'
 import smallLogo from '../../images/small_logo.png'
 import { MEDIUM_WIDTH, MOBILE_WIDTH } from '../../constants'
 import './Footer.css'
+import { Link } from 'react-router-dom'
 
-function sitemapLink(link, legend, big) {
-  if (big) {
-    return (
-      <span>
-        <a href={link} className="sitemap-link">
-          { legend }
-        </a>
-        <br />
-      </span>
-    )
-  }
-  return (
-    <a href={link} className="sitemap-link">
-      { legend }
-    </a>
-  )
-}
-
-function sitemapIconButton(link, icon) {
-  return (
-    <a
-      href={link}
-      className="icon-link"
-    >
-      { icon }
-    </a>
-  )
-}
+type RouteType = {
+  path: string,
+  legend: string,
+  component: string
+};
 
-class Footer extends Component {
-  constructor(props) {
-    super(props)
+type Props = {
+  routes: Array<RouteType>
+};
 
-    this.setSizeAtributes = this.setSizeAtributes.bind(this)
-    this.largeView = this.largeView.bind(this)
-    this.smallView = this.smallView.bind(this)
-    this.goUp = this.goUp.bind(this)
+type State = {
+  icon_size: number,
+  view_size_large: boolean
+};
 
-    this.members = props.members
+class Footer extends Component<Props, State> {
+  constructor(props : Props) {
+    super(props)
 
     this.state = {
       icon_size: (window.innerWidth >= MOBILE_WIDTH) ? 40 : 35,
       view_size_large: (window.innerWidth > MEDIUM_WIDTH),
     }
   }
 
-  componentDidMount() {
-    window.addEventListener('resize', this.setSizeAtributes)
-  }
-
-  goUp() {
-    window.scrollTo(0, 0)
-  }
-
-  setSizeAtributes() {
+  setSizeAtributes = () => {
     this.setState({
       icon_size: (window.innerWidth >= MOBILE_WIDTH) ? 40 : 35,
       view_size_large: (window.innerWidth > MEDIUM_WIDTH),
     })
   }
 
+  goUp = () => {
```

Comment:
Can you change this to a more descriptive name?
```suggestion
  scrollToTop = () => {
```